JetStreamDriver.js

```diff
@@ -2258,25 +2263,28 @@ for (const benchmark of BENCHMARKS) {
 
 this.JetStream = new Driver();
 
-function enableBenchmarksByName(testName)
+function enableBenchmarksByName(testName, forceEnable)
 {
     const benchmark = benchmarksByName.get(testName);
 
-    if (benchmark)
-        JetStream.addBenchmark(benchmark);
-    else
+    if (!benchmark)
         throw "Couldn't find test named \"" +  testName + "\"";
+
+    if (!forceEnable && benchmark.disabledByDefault)
```

Comment:
nit: would we ever not forceEnable workloads?

Reply:
We don't but I had the argument to be consistent. I'll remove.